src/mcp_atlassian/utils/environment.py

```diff
@@ -69,6 +80,17 @@ def get_available_services() -> dict[str, bool | None]:
             logger.info(
                 "Using Jira OAuth 2.0 (3LO) authentication (Cloud-only features)"
             )
+        elif all(
+            [
+                os.getenv("ATLASSIAN_OAUTH_ACCESS_TOKEN"),
+                os.getenv("ATLASSIAN_OAUTH_CLOUD_ID"),
+            ]
+        ):
+            jira_is_setup = True
+            logger.info(
+                "Using Confluence OAuth 2.0 (3LO) authentication (Cloud-only features) "
```

Comment:
Copy-paste error in log message
```suggestion
                "Using Jira OAuth 2.0 (3LO) authentication (Cloud-only features) "
```

Reply:
Fixed!